orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/gce/GoogleSecurityGroupUpserter.groovy

```diff
@@ -41,8 +41,16 @@ class GoogleSecurityGroupUpserter implements SecurityGroupUpserter, CloudProvide
   @Override
   SecurityGroupUpserter.OperationContext getOperationContext(Stage stage) {
     def ops = [[(SecurityGroupUpserter.OPERATION): stage.context]]
+
+    // We used to get the `name` directly from the `context` which works
+    // tasks but not pipelines because orca will remove `name` from the context
+    // before it passes it on to stages/tasks.
+    // We switch to using `securityGroupName` field but for backwards compat we will support
+    // both for while all services deploy and queues drain with old field
+    String securityGroupName = stage.context.securityGroupName ?: stage.context.name
+
     def targets = [
-        new MortService.SecurityGroup(name: stage.context.name,
+        new MortService.SecurityGroup(name: securityGroupName,
```

Comment:
I wonder who is this "Mort" guy?

Reply:
`mort` was one of the services that existed pre-clouddriver ... it was responsible for things like security groups, vpcs, accounts ...

mort + oort + kato == clouddriver

Reply:
👍